server/test/factories.ts

```diff
@@ -336,7 +340,7 @@ export async function buildDocument(
     overrides.userId = user.id;
   }
 
-  if (!overrides.collectionId) {
+  if (!draft && !overrides.collectionId) {
```

Comment:
We can probably change this to check that `collectionId` isn't explicitly set as `null` then.

```
buildDraftDocument(overrides) {
  return buildDocument({ ...overrides, collectionId: null, publishedAt: null });
}
`